deploy/examples/operator.yaml

```diff
@@ -162,9 +162,9 @@ data:
   # NOTE: Support for defining NodeAffinity for operators other than "In" and "Exists" requires the user to input a
   # valid v1.NodeAffinity JSON or YAML string. For example, the following is valid YAML v1.NodeAffinity:
   # CSI_CEPHFS_PLUGIN_NODE_AFFINITY: |
-  # requiredDuringSchedulingIgnoredDuringExecution:
-  #   nodeSelectorTerms:
-  #     - matchExpressions:
+  #   requiredDuringSchedulingIgnoredDuringExecution:
+  #     nodeSelectorTerms:
+  #       - matchExpressions:
   #         - key: myKey
```

Comment:
I found out that the indentation was indeed a bit off here (4 spaces instead of 2), fixed and tested.